lib/rocket_pants/controller/instrumentation.rb

```diff
@@ -8,17 +8,17 @@ module Instrumentation
 
     def process_action(action, *args)
       raw_payload = {
-        :controller => self.class.name,
-        :action     => self.action_name,
-        :params     => request.filtered_parameters,
-        :formats    => [:json],
-        :method     => request.method,
-        :path       => (request.fullpath rescue "unknown")
+          :controller => self.class.name,
+          :action     => self.action_name,
+          :params     => request.filtered_parameters,
+          :formats    => [:json],
+          :method     => request.method,
+          :path       => (request.fullpath rescue "unknown")
       }
 
-      ActiveSupport::Notifications.instrument("start_processing.rocket_pants", raw_payload.dup)
+      ActiveSupport::Notifications.instrument("start_processing.action_controller", raw_payload.dup)
 
-      ActiveSupport::Notifications.instrument("process_action.rocket_pants", raw_payload) do |payload|
+      ActiveSupport::Notifications.instrument("process_action.action_controller", raw_payload) do |payload|
```

Comment:
Why change these to the word `action_controller`? Seems like we'd want these to remain `rocket_pants` since that's where the notification is coming from?

Reply:
I'll defer to Curtis for the explanation, but if it's rocket_pants we can't use Skylight instrumentation -- we're currently patching rocket_pants in the monorail to change this to action_controller